decidim-proposals/spec/helpers/application_helper_spec.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+require "spec_helper"
+
+module Decidim
+  module Proposals
+    describe ApplicationHelper do
+      describe "humanize_proposal_state" do
+        let(:helper) do
```

Comment:
I don't know why but rspec goes crazy when writing specs for an application controller. It happened a while ago too.